Harvest.Web/ClientApp/src/App.tsx

```diff
@@ -30,30 +30,33 @@ function App() {
     <AppContext.Provider value={Harvest}>
       <Route exact path="/" component={Home} />
       <Route exact path="/home/spa" component={Spa} />
-      <Route path="/request/create/:projectId?" component={RequestContainer} />
-      <Route path="/request/approve/:projectId" component={ApprovalContainer} />
-      <Route
+      <ConditionalRoute roles={["FieldManager"]} path="/request/create/:projectId?" component={RequestContainer} />
```

Comment:
why can't non-FMs create a project?  shouldn't everyone be able to?

Reply:
Fixed